yarn-project/end-to-end/src/e2e_fees/gas_estimation.test.ts

```diff
@@ -96,7 +96,7 @@ describe('e2e_fees gas_estimation', () => {
 
   it('estimates gas with public payment method', async () => {
     // TODO(#11324): Reset this value back to zero.
-    const estimatedGasPadding = 0.00068359375;
+    const estimatedGasPadding = 0.1;
```

Comment:
Not sure why the estimation became even more incorrect. But as mentioned in [the issue](https://github.com/AztecProtocol/aztec-packages/issues/11324) will revisit this once we have a note packing (that will be done in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/11376)).